frontend/src/metabase/containers/UndoListing.styled.tsx

```diff
@@ -35,6 +35,14 @@ export const CardContentSide = styled.div`
   align-items: center;
 `;
 
+export const TruncatedText = styled.div`
```

Comment:
I tried to refactor this to use Ellipsified but it wasn't "getting smaller" anymore when the viewport was small:
<img width="817" alt="image" src="https://github.com/metabase/metabase/assets/1914270/bcf8bddf-1f06-4dc9-b6a7-ad4a76db71b1">

That seems to be related to the css that is used when lines==1

https://github.com/metabase/metabase/blob/4f53f052867a5f5cf7c6853353a779d6ec44917c/frontend/src/metabase/core/components/Ellipsified/Ellipsified.styled.tsx#L4-L8

Reply:
Thanks @deniskaber , had to tweak it a bit but it's working now